pages/index.html

```diff
@@ -4,9 +4,26 @@
   <meta charset="UTF-8">
```

Comment:
We should bring the index.html out of the pages directory and have it live in the root instead. If the index.html is in the root of the project then it would open as the home page.